addusers.py

```diff
@@ -417,10 +419,19 @@ def parse_rows(rows, select_user=None):
         for user in content[project].users:
             ks_user = exists_in_keystone(user, ks_users)
             try:
+                if len(rows[content[project].row]) != len(rows[0]):
+                    extend_blank_space = len(rows[0]) -\
+                        len(rows[content[project].row])
+                    blank_space = [''] * extend_blank_space
+                    rows[content[project].row].extend(blank_space)
                 if user.is_new:
                     if ks_user:
                         raise ItemExistsError('User', user.name)
                     new_ks_user = openstack.create_user(user, ks_project)
+                    rows[content[project].row].extend(
+                        [new_ks_user.id,
+                         datetime.datetime.now().strftime(
```

Comment:
I wonder if we shouldn't do `datetime.datetime.now()` just once and store the result, so that all rows processed in a single run of the script get an identical timestamp?  I do this with the other timestamps.  It's not wrong to do it the way you did it here, more of a philosophical question.